13/alpine/Dockerfile

```diff
@@ -1,18 +1,18 @@
 FROM alpine:3.10
 
-ENV NODE_VERSION 13.0.1
+ENV NODE_VERSION 13.1.0
 
 RUN addgroup -g 1000 node \
     && adduser -u 1000 -G node -s /bin/sh -D node \
     && apk add --no-cache \
         libstdc++ \
     && apk add --no-cache --virtual .build-deps \
         curl \
-    && ARCH= && alpineArch="$(arch)" \
+    && ARCH= && alpineArch="$(apk --print-arch)" \
       && case "${alpineArch##*-}" in \
         x86_64) \
           ARCH='x64' \
-          CHECKSUM="5f40392e14e81a566c06f4abcbe735575f2d7045709211eb7245ef8f67896bee" \
```

Comment:
not sure about this. should I manually roll it back?

Reply:
Did you get an error when running `update.sh`? Maybe try running it again?

Reply:
nope, no errors. I can try to rerun

Reply:
OK, it showed up now